real/wxpro/spec.yml

```diff
@@ -0,0 +1,25 @@
+architectures:
+  - x86_64
+categories:
+  - real_world
+  - issta_2017
+  - aachen
+dependencies:
+  cmath: {}
+language: gnu99
+name: wxpro
+schema_version: 0
+sources:
+  - main.c
+  - wxpro/units.c
```

Comment:
@erzett You're adding a lot of new source files but you only have two here. Is that really all that's needed? Is there no library to build?